### PR TITLE
chore: update sidebar links

### PR DESCRIPTION
**Pull request by achrinza**
Signed-off-by: Rifa Achrinza <25147899+achrinza@users.noreply.github.com>

### CODE REVIEW

**Comment by achrinza** [comment]
Commit Linter seems to be failing on the commit sign-off.

**Comment by dhmlau** [comment]
@achrinza, the commit linter was complaining about:
```
**************************************************
**
**  Linting commit logs
**
**  1 problems found:
**    730fce6 - chore: update sidebar links: line 3 longer than 72 characters.
**
**************************************************
```

**Comment by achrinza** [comment]
@dhmlau Thanks for looking into it. Line 3 is the commit sign-off which I believe is required for the DCO check.

See https://github.com/strongloop/loopback-next/issues/4492

**Comment by dhmlau** [comment]
Not sure if the "line 3" it complains about is the blank line or the DCO sign off message, the DCO message indeed has more than 72 characters. 
@strongloop/loopback-maintainers, since the max number of characters for LB4 commits is 100 (see https://loopback.io/doc/en/lb4/code-contrib-lb4.html#body), does it make sense to relax this limit? Thanks.

**Comment by dhmlau** [comment]
Discussed with @bajtos, since there won't be much updates in this repo, we can force merge the PR with the commit linter error when it's ready.

File: index.loopback.html [hunk 1]
```diff
@@ -165,21 +165,21 @@
   <div class="container">
       <div class="row text-center">
           <div class="col-md-3">
-              <img src="https://v4.loopback.io/img/global/loopback-mark-frame-white.svg" width=40px/>
+              <img src="https://loopback.io/img/global/loopback-mark-frame-white.svg" width=40px/>
```
Comment (dhmlau) [inline index.loopback.html]: It seems to be a broken link for the svg.

File: index.loopback.html [hunk 2]
```diff
@@ -165,21 +165,21 @@
...
           </div>
           <div class="col-md-3">
               <p class="label text-uppercase">Product</p>
               <p class="regular">
-                  <a href="http://loopback.io/doc/en/lb4/Getting-started.html">Getting Started</a>
+                  <a href="https://loopback.io/doc/en/lb4/Getting-started.html">Getting Started</a>
               </p>
               <p class="regular">
-                  <a href="http://loopback.io/doc/en/lb4/Examples-and-tutorials.html">Tutorials</a>
+                  <a href="https://loopback.io/doc/en/lb4/Examples-and-tutorials.html">Tutorials</a>
```
Comment (dhmlau) [inline index.loopback.html]: This link no longer exist. At some point, we split the examples and tutorials into 2 pages, i.e. 
https://loopback.io/doc/en/lb4/Tutorials.html
https://loopback.io/doc/en/lb4/Examples.html

**Review by dhmlau — CHANGES_REQUESTED** [review]
@achrinza, thanks for updating the html not using v4.loopback.io.  I noticed there are other references of `v4.loopback.io`.  Since you're updating it, could you please also take care of those too? Thanks.

**Comment by achrinza** [comment]
@dhmlau Everything should be fixed

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
Thanks @achrinza! I'm going to force merge it because the commit linter is due to the DCO signing message.

**Pull request merged**